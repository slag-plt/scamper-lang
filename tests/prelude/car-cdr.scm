(car (cons #t #f))
(car (cons 1 2))
(car (cons "hi" "bye"))
(car (cons "a" "b"))
(car (cons 0.003 100))
(cdr (cons 1 2))
(cdr (cons #t #f))
(cdr (cons "hi" "bye"))
(cdr (cons "a" "b"))
(cdr (cons 0.003 100))
