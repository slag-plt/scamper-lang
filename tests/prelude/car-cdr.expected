#t
1
"hi"
"a"
0.003
2
#f
"bye"
"b"
100
